src/Core/Components/Button/FluentSplitButton.razor.cs

```diff
@@ -0,0 +1,150 @@
+        .Build();
+
+    /// <summary />
+    protected string? StyleValue => DefaultStyleBuilder
+        .AddStyle("background-color", BackgroundColor, when: () => !string.IsNullOrEmpty(BackgroundColor))
+        .AddStyle("color", Color, when: () => !string.IsNullOrEmpty(Color))
+        .Build();
+
+    /// <summary>
+    /// Gets or sets the shape of the button.
+    /// The default value is `null`. Internally the component uses <see cref="ButtonShape.Rounded"/> as its default value.
+    /// </summary>
+    [Parameter]
+    public ButtonShape? Shape { get; set; }
+
+    /// <summary>
+    /// Gets or sets the size of the button.
+    /// The default value is `null`. Internally the component uses <see cref="ButtonSize.Medium"/> as its default value.
+    /// </summary>
+    [Parameter]
+    public ButtonSize? Size { get; set; }
+
+    /// <summary>
+    /// Gets or sets the name of the element.
+    /// Allows access by name from the associated form.
+    /// </summary>
+    [Parameter]
+    public string? Name { get; set; }
+
+    /// <summary>
+    /// Gets or sets the visual appearance.
+    /// The default value is `null`. Internally the component uses <see cref="ButtonAppearance.Outline"/> as its default value.
+    /// </summary>
+    [Parameter]
+    public ButtonAppearance? Appearance { get; set; }
+
+    /// <summary>
+    /// Gets or sets the background color of this button (overrides the <see cref="Appearance"/> property).
+    /// Set the value `rgba(0, 0, 0, 0)` to display a transparent button.
+    /// </summary>
+    [Parameter]
+    public string? BackgroundColor { get; set; }
+
+    /// <summary>
+    /// Gets or sets the color of the font (overrides the <see cref="Appearance"/> property).
+    /// </summary>
+    [Parameter]
+    public string? Color { get; set; }
+
+    /// <summary>
+    /// Gets or sets if the button only shows an icon
+    /// Can be used when using <see cref="ChildContent"/> that renders as an icon
+    /// </summary>
+    [Parameter]
+    public bool IconOnly { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Icon"/> displayed at the start of button content.
+    /// </summary>
+    [Parameter]
+    public Icon? IconStart { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Icon"/> displayed at the end of button content.
+    /// </summary>
+    [Parameter]
+    public Icon? IconToggle { get; set; }
+
+    /// <summary>
+    /// Gets or sets the title of the button.
+    /// The text usually displayed in a `tooltip` popup when the mouse is over the button.
+    /// </summary>
+    [Parameter]
+    public string? Title { get; set; }
+
+    /// <summary>
+    ///  Gets or sets the content to be rendered inside the button.
+    ///  This can be used as an alternative to specifying the content as a child component of the button.
+    ///  If both are specified, both will be rendered.
+    ///  </summary>
+    [Parameter]
+    public string? Label { get; set; }
```

Comment:
Should we have a `LabelTemplate` to help customization?

Reply:
No. In this case the Label is the actual text displayed on the button. We can't use ChildContent for this in this case as that holds the menu items

Reply:
It is only for the text displayed on the button. We could add it later but for now it is not needed I think. We do not have any example where the label on a button is using formattted (HTML) text